client/layout/masterbar/drafts.jsx

```diff
@@ -0,0 +1,133 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSite } from 'state/ui/selectors';
+import Popover from 'components/popover';
+import Count from 'components/count';
+import { getMyPostCounts } from 'state/posts/counts/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import {
+	getSitePostsForQueryIgnoringPage,
+	isRequestingSitePostsForQuery
+} from 'state/posts/selectors';
+import Draft from 'my-sites/draft';
+import QueryPosts from 'components/data/query-posts';
+import QueryPostCounts from 'components/data/query-post-counts';
+import Button from 'components/button';
+import Site from 'blocks/site';
+import { getCurrentUserId } from 'state/current-user/selectors';
+import sitesList from 'lib/sites-list';
```

Comment:
> We need to clean up Draft, which I prefer not to do in this pass.

OK, but since we're adding _new_ code that uses an anti-pattern, we should still comment and explain why.